docs/docs/administration/08-advanced/01-external-authentication.md

```
@@ -126,15 +126,82 @@ To disable the shibboleth session check you can set the following `.env` variabl
+
+    The application authenticates with the token endpoint using the `client_secret_basic` method.
+
+- ##### Supported Signing Algorithms
+    - RS256, RS384, RS512
+    - PS256, PS384, PS512
+    - ES256, ES384, ES512
+    - EdDSA (only with the Ed25519 curve)
+    - HS256, HS384, HS512
+
+- ##### Limitations
+    - Encrypted ID token responses are **not supported**.
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)